server/project/parse.ts

```diff
@@ -77,3 +78,40 @@ export function parseRouterFunctions(code: string) {
 
   return routerFunctions;
 }
+
+export function createOperation(code: string) {
```

Comment:
Now I look back, the word `operation` and the word `route` in the same codebase doesn't play together well. Let's just stick to `route`? If following comments above this would become `generateRoutesForTestingClient`. We can do that renaming in the frontend code as well.